README.md

```diff
@@ -1,5 +1,3 @@
-[ci-img]: https://img.shields.io/travis/ciena-frost/ember-frost-select.svg "Travis CI Build Status"
-[ci-url]: https://travis-ci.org/ciena-frost/ember-frost-select
```

Comment:
Nobody noticed I accidentally removed these lines from the `README.md`? :disappointed: